routes/api.php

```diff
@@ -130,20 +122,16 @@
     Route::get(
         '/jam/playlist',
         function () {
-        if (Cache::has('jam') === false) {
-            return response()->json(['message' => 'No one be jammin\''], Response::HTTP_SERVICE_UNAVAILABLE);
-        }
+            if (Cache::has('jam') === false) {
+                return response()->json(['message' => 'No one be jammin\''], Response::HTTP_SERVICE_UNAVAILABLE);
+            }
```

Comment:
I think this can be replaced with applying the middleware, as you did elsewhere.